dist/libblockdev.spec.in

```diff
@@ -296,6 +296,7 @@ Summary:     The partitioning plugin for the libblockdev library
 Requires: %{name}-utils%{?_isa} >= 0.11
 Requires: device-mapper-multipath
 Requires: gdisk
+Requires: util-linux
```

Comment:
The *part* plugin should also test the availability of the ``sfdisk`` utility in its ``bd_part_check()`` function.